src/main/java/com/salesforce/ConsumeTopic.java

```diff
@@ -110,7 +112,6 @@ public Exception call() {
                 String truncatedValue = lastValue.length() <= 15 ? lastValue : lastValue.substring(0, 15);
                 log.debug("Last consumed message {} -> {}..., consumed {} messages, topic: {}",
                         lastMessage.key(), truncatedValue, messages.count(), topicName);
-                Thread.sleep(readWriteInterval);
```

Comment:
Why not set the config to zero? Or split the configs to "read interval" and "write interval"?